### ISSUE TITLE
Some questions about the experiment on the CelebA dataset

### ISSUE THREAD

**Issue by jfzhuang** [JiahuiYu/generative_inpainting#86]
I have conducted a experiment on the CelebA dataset, With randomly spliting the training subset(199399 samples) and testing subset(3200 samples), I have trained the model from scratch with 500k iteration. However, I can not obtain a satisfactory result.
![606620164751274400](https://user-images.githubusercontent.com/20696784/42731403-356a7016-883f-11e8-8ddb-6a104ca9d986.jpg)
![195713833015263495](https://user-images.githubusercontent.com/20696784/42731404-37c74e7e-883f-11e8-872d-cf0f5ff59e8c.jpg)
I wonder if you can provide more training configurations on the CeleA dataset, such as training/testing split, total iteration and other related trianing hyperparameters. Thank you a lot!

**Comment by JiahuiYu** [comment]
Hi, the training configuration is exactly same as it is in the code. Split is the default split of celebA. Training is done in 5 days with one 1080 Ti. I guess you need to train for more iterations.

**Comment by Gavin666Github** [comment]
hey,jiahui,here are images during my training on CeleA dataset .
![image](https://user-images.githubusercontent.com/35061171/43193950-d03db49e-9033-11e8-9686-7a8617f3e210.png)

I don't know when I can stop training, and I want to ask you how many epochs you trained. Thx.

**Comment by JiahuiYu** [comment]
@Gavin666Github Hi, the results still have many artifacts. Have you figure out why? Did you look at the convergence curves of WGAN in tensorboard? Basically I trained for 5 days on one 1080Ti GPU.

**Comment by JiahuiYu** [comment]
@Gavin666Github Hi I guess you will need to train more epochs because `g_loss` is still increasing. The WGAN loss should converge at the end of training. You may also be interested in reading paper of [WGAN](
https://arxiv.org/abs/1701.07875) and [WGAN-GP](
https://arxiv.org/abs/1704.00028).

**Comment by Gavin666Github** [comment]
@JiahuiYu OK, thank you very much. I will read the articles you recommended carefully.

**Comment by godchengzhihang** [comment]
@Gavin666Github Hello, when you are training, is the print loss value always decreasing? Why did my first decrease and then increase?

**Comment by JiahuiYu** [comment]
@godchengzhihang Hi it is normal for training GANs.
